book.rb

```diff
@@ -0,0 +1,15 @@
+# class book
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
```

Comment:
- [ ] Kindly make the `@rental` have _**setter**_ and _**getter**_ methods: `attr_accessor :rentals`